x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysTablesTests.java

```diff
@@ -353,7 +353,7 @@ private SqlTypedParamValue param(Object value) {
             SqlTestUtils.TEST_CFG,
             new FunctionRegistry(),
             IndexResolution.valid(test),
-            new Verifier(new Metrics())
+            new Verifier(new Metrics(), SqlTestUtils.TEST_CFG.version())
```

Comment:
All these tiny changes to tests classes to introduce the version in the constructor shouldn't, hopefully, be needed anymore.

Reply:
Indeed, reverted these changes.